bootstrap.sh

```diff
@@ -9,10 +9,10 @@ echo Building the build tool...
 GO111MODULE=on go get ./util/src/harvey/cmd/...
 
 echo Fetching u-root and building it...
-GO111MODULE=on go get github.com/u-root/u-root
+GO111MODULE=on go get github.com/u-root/u-root@c6181ee270c59d542825e89a7c02c8c25b30373b
 
 echo Fetch harvey-os.org commands and build them into $GOBIN
-GO111MODULE=on go get harvey-os.org/cmd/...
+GO111MODULE=on go get harvey-os.org@bcfa7228c80c35fc627c077fdd5918db2181489e
```

Comment:
Why is `/cmd/...` remove? There is no Go package at the top-level directory.
```suggestion
GO111MODULE=on go get harvey-os.org/cmd/...@bcfa7228c80c35fc627c077fdd5918db2181489e
```